<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_detail_ajout</class>
 <widget class="QDialog" name="C_detail_ajout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>230</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="btn_enregistrer_detail">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ajouter_detail">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_titre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_mondial">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Local:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_film">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre Film</string>
   </property>
  </widget>
  <widget class="QLabel" name="BLOCKED_txt">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID mondial:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
